Xamarin.Forms.Core/Shapes/GeometryGroup.cs

```diff
@@ -25,5 +35,67 @@ public FillRule FillRule
             set { SetValue(FillRuleProperty, value); }
             get { return (FillRule)GetValue(FillRuleProperty); }
         }
+
+        public event EventHandler InvalidateGeometryRequested;
+
+        void UpdateChildren(GeometryCollection oldCollection, GeometryCollection newCollection)
```

Comment:
Think you need a quick reformat here - it's using spaces instead of tabs.

Reply:
Fixed!